### ISSUE TITLE
Overfitting using VGG?

### ISSUE THREAD

**Issue by LeslieMei** [clovaai/voxceleb_trainer#38]
Hello, I trained with the VGG40 network, the results and setting of hyper parameters are in scores.txt. 
The EER and TEER(which I think is accuracy) after 10 epochs is 10.67% and 71.71% respectively.
The TEER becomes higher with the epochs increasing, but the EER is 12.02% after 20 epochs and 12.58% after 30 epochs.
Does this means overfitting?
[scores.txt](https://github.com/clovaai/voxceleb_trainer/files/4897159/scores.txt)

**Comment by joonson** [comment]
TEER in the case of some losses actually means Top-1 train accuracy. So higher is better.

**Comment by LeslieMei** [comment]
Thank you very much for your reply.
As the higher accuracy means the model is better, I think the EER should be lower with the training going. I don't know why the Equal Error Rate (evaluating every 10 epochs) becomes higher.

**Comment by muzihuole** [comment]
Can you provide me the pretrained model of VGGVox. Looking forward to your reply.
